vmc/resource_vmc_sddc.go

```diff
@@ -364,8 +367,47 @@ func resourceSddcUpdate(d *schema.ResourceData, m interface{}) error {
 	sddcID := d.Id()
 	orgID := (m.(*ConnectorWrapper)).OrgID
 
+	// Convert sddc from 1node to default
+	toConvertSddc := false
+	if d.HasChange("sddc_type") {
+		oldTmp, newTmp := d.GetChange("sddc_type")
+		oldType := oldTmp.(string)
+		newType := newTmp.(string)
+
+		// Validate for convert type params
+		if oldType == "1NODE" && (newType == "" || newType == "DEFAULT") {
+			oldTmp, newTmp := d.GetChange("num_host")
```

Comment:
oldTmp should be d.Get("num_host").(int) right ?

Reply:
I will change it like your next comments. Since this format is the same as we used in sddcUpdate.